contents/customers/swype.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 'How Swype uses mobile session replays for testing, user insights'
+customer: Great Expectations
+logo: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/swype_34c89913d9.png
+logoDark: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/swype_logo_dark_acb437c1ef.png
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog_great_expectations_f90a29aa4d.jpg
+featuredCustomer: false
+industries:
+  - 'Recruitment, Data'
+users:
+  - Founder
+  - Engineering
+toolsUsed:
+  - Session replay
+  - Mobile
+  - Insights
+date: 2024-10-22
+---
+
+Built in just over a month, [Swype](https://swype.jobs/) is an iOS app that helps people find new jobs using a tailored AI agent. Like Tinder for jobs, candidates can swipe left and right to dismiss or automatically apply for interesting roles — and the team has built PostHog into the app from the start to get the insights they need. 
+
+“We were considering options like Firebase Analytics or building our own analytics,” says Swipe’s founding full-stack engineer, David Alade. “We thought about either sending messages to our backend, or using something like Sentry to send logs from the app — but we settled on PostHog because of how easy it was to set everything up and present the data.”
+
+Analytics was an early need, but it wasn’t the only tool the team was interested in. Swype’s app has such a streamlined and simplified interface that they quickly realized how challenging it would be to extrapolate much knowledge from event tracking alone. They needed another way to observe user behavior in a more holistic and intuitive way.
+
+“The thing about mobile app development is you want to really reduce complexity and the number of buttons per page,” explains David. “But then it’s hard to spot in the data alone when you’re so close to what you’re building. Session replays are great because you can just _see_ what’s happening: I can see them clicking things again and again in frustration.”
+
+Tempted by the monthly allowance of 5,000 free replays, David quickly deployed PostHog mobile replays using [the iOS SDK](/docs/libraries/ios) and using it alongside product
+
+### One reason why PostHog is better than Netflix
+
+Since deploying mobile replays David has made it a habit to watch as many recordings as he can — sometimes even watching them in an uninterrupted stream when he doesn’t have a particular event he wants to filter by. 
```

Comment:
```suggestion
Since deploying mobile replays, David watches as many recordings as he can — sometimes even watching them in an uninterrupted stream when he doesn’t have a particular event he wants to filter by. 
```